cli/crud/import/containers.go

```diff
@@ -0,0 +1,163 @@
+// This file contains types for defining containers (structs) that store
+// contexts during crud import work.
+
+package crudimport
+
+import (
+	"bufio"
+	"os"
+)
+
+// importSummaryСontainer contains information for the import summary.
+type importSummaryСontainer struct {
+	// readTotal is counter of total iterations of the parser.
+	readTotal uint32
+	// ignoredDueToProgress is counter of skipped iterations of the parser due to progress file.
+	ignoredDueToProgress uint32
+	// parsedSuccess is counter of succsess iterations of the parser.
+	parsedSuccess uint32
+	// parsedError is counter of fail iterations of the parser.
+	parsedError uint32
+	// importedSuccess is counter of succsess iterations of crud stored procedure.
+	importedSuccess uint32
+	// importedError is counter of fail iterations of crud stored procedure.
+	importedError uint32
+}
+
+// Opts contains information about options from the user via the CLI.
+type Opts struct {
+	// ConnectUsername is username using for establish connection to router via tt connect.
+	ConnectUsername string
+	// ConnectPassword is password using for establish connection to router via tt connect.
+	ConnectPassword string
+	// Format is input file format.
+	Format string
+	// Header indicates the presence of a header in input file.
+	Header bool
+	// LogFileName is name of log file with logs about unsuccessful imported data.
+	LogFileName string
+	// ErrorFileName is name of error file with unsuccessful imported data.
+	ErrorFileName string
+	// SuccessFileName is name of success file with successful imported data.
+	SuccessFileName string
+	// Match sets the match rule between input fields and space fields.
+	Match string
+	// BatchSize is tuples amount within batch.
+	BatchSize uint32
+	// Progress is flag for using the progress file from previous launch.
+	Progress bool
+	// Operation sets type of crud stored procedures during import (inser of replace).
+	Operation string
+	// OnError sets the action when an error is detected (stop or skip).
+	OnError string
+	// NullVal sets null interpretation for input data.
+	NullVal string
+}
+
+// dumpSubsystemFdСontainer contains file descriptors for the disk dump subsystem.
+type dumpSubsystemFdСontainer struct {
+	// logFile is file descriptor of file with logs about unsuccessful imported data.
+	logFile *os.File
+	// errorFile is file descriptor of file with unsuccessful imported data.
+	errorFile *os.File
+	// successFile is file descriptor of file with successful imported data.
+	successFile *os.File
+	// progressFile is file descriptor of file with progress information.
+	progressFile *os.File
+}
+
+// parserCtxСontainer contains a part of the tuple context that related to the parser.
```

Comment:
I think it doesn't matter what the structure is a part of, what matters is what it describes.